src/seedu/addressbook/AddressBook.java

```diff
@@ -205,21 +205,7 @@
 	 */
```

Comment:
Ensure you update the test script to include test cases for this new functionality. add both positive and negative test cases.